Machine_Learning/notebooks/czmodel/Regresssion_PyTorch_5_0_0.ipynb

```diff
@@ -0,0 +1,676 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "cHZX_QBJAsxR",
+    "outputId": "81d368f6-4b14-479c-cf2f-12574f460fe9",
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Install czmodel and dependencies\n",
+    "! pip install --upgrade pip\n",
+    "! pip install \"czmodel>=5.0,<6.0\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "XCOyzv2frEEE"
+   },
+   "outputs": [],
+   "source": [
+    "# This can be used to switch on/off warnings\n",
+    "import warnings\n",
+    "warnings.filterwarnings('ignore')\n",
+    "warnings.simplefilter('ignore')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "The library CZModel provides a base package and extras for export functionalities that require specific dependencies -:\n",
```

Comment:
Maybe add a link here:
```suggestion
    "The library [CZModel](https://pypi.org/project/czmodel/) provides a base package and extras for export functionalities that require specific dependencies -:\n",
```

Reply:
Thanks for the comment! I've added the links to all the notebooks.